src/serenity/serenity.hpp

```diff
@@ -9,6 +9,9 @@
 #include "stout/nothing.hpp"
 #include "stout/try.hpp"
 
+// TODO(bplotka): Uncomment that when using Mesos > 0.25.0-rc2
+// #define NEWEST_LIBPROCESS
```

Comment:
This should be an option inside cmakefile.
Also, it should be called differently - eg. MESOS_BELLOW_0_25_0 (also, I reversed logic), because here we have strict dependency on Mesos version (we share same libs). 
MESOS_BELLOW because IMO we should strive for newest Mesos support.